README.md

```diff
@@ -0,0 +1,47 @@
+# OpenAssetIO Test Host
+
+A bare-bones C++ host application with [OpenAssetIO](https://github.com/OpenAssetIO/OpenAssetIO)
+as a dependency, for testing basic integration (linking, headers, etc).
+
+Test targets are created that simply check that C++, C and
+Python components can be imported.
+
+> **Note**
+>
+> This project is intended for use in CI and not used as a reference.
+
+## Getting started
+
+This application can link against OpenAssetIO in two modes, depending on
+the value of the `OPENASSETIOTEST_ENABLE_SUBPROJECT` CMake variable -
+see below.
+
+The usual CMake steps can be used to build and run the tests. Assuming
+the current working directory is at the root of the repository:
+
+```shell
+cmake -S . -B build
+cmake --build build
+ctest --test-dir build
+```
+
+### `OPENASSETIOTEST_ENABLE_SUBPROJECT=OFF` (default)
+
+When `OPENASSETIOTEST_ENABLE_SUBPROJECT=OFF` (which is the default if
+unspecified), OpenAssetIO will be discovered as a CMake package
+(`find_package(OpenAssetIO)`) and so OpenAssetIO must be pre-built and
+installed in a discoverable location.
```

Comment:
Worth being explicit here, and say, add the installed location to CMAKE_PREFIX_PATH?

Reply:
I will add that as an example, but of course there are many alternatives to `CMAKE_PREFIX_PATH`.

Reply:
Well, that was also when I was thinking only config files were being used.

How about saying "in a discoverable location matching the documented find_package search options" (or similar)?

Reply:
Does this read a bit better: 92b5bd49 ?